kubeflow/common/centraldashboard.libsonnet

```diff
@@ -155,12 +155,67 @@
     },  // role binding
     centralDashboardRoleBinding:: centralDashboardRoleBinding,
 
+    local centralDashboardClusterRole = {
+      apiVersion: "rbac.authorization.k8s.io/v1",
+      kind: "ClusterRole",
+      metadata: {
+        labels: {
+          app: "centraldashboard",
+        },
+        name: "centraldashboard",
+        namespace: params.namespace,
```

Comment:
We don't need a namespace for ClusterRole as it's scope cross namespaces.

Reply:
Change made